### DISCUSSION TITLE
Universal CRT status relationship

### DISCUSSION THREAD

**Discussion by tumagonx** [microsoft/STL#2722] [General] [answered]
Off-Topic 

With STL require Universal CRT, will Microsoft plan to open source it as well?

**Comment by tumagonx** [comment]
I mean in the same license as STL, I knew the CRT is "source-available"...

**Comment by frederick-vs-ja** [comment]
Also ask: are there some 3rd party codes in UCRT that can't be open-sourced at this moment?

**Comment by cbezault** [comment]
@amyw-msft for a definitive answer. But I presume in no uncertain terms that there is no intention to open source the UCRT.

**Comment by amyw-msft** [comment, marked as answer]
There are currently no plans to open source the UCRT. Yes, the UCRT does contain code that is licensed, which does complicate open sourcing. These portions are also not "source-available" via the WinSDK.
